src/commands/docker.ts

```diff
@@ -0,0 +1,129 @@
+import * as Sentry from "@sentry/node";
+import { CancellationToken, Disposable, ProgressLocation, Uri, env, window } from "vscode";
+import { registerCommandWithLogging } from ".";
+import { getLocalKafkaImageName, isDockerAvailable } from "../docker/configs";
+import { LocalResourceWorkflow } from "../docker/workflows";
+import { ConfluentLocalWorkflow } from "../docker/workflows/confluent-local";
+import { Logger } from "../logging";
+
+const logger = new Logger("commands.docker");
+
+async function startLocalResourcesWithProgress() {
+  await runWorkflowWithProgress();
+}
+
+/** Run the local resource workflow(s) with a progress notification. */
+async function runWorkflowWithProgress(start: boolean = true) {
```

Comment:
Not really necessary to split into two functions this PR, but makes more sense starting in https://github.com/confluentinc/vscode/pull/446